test/dev/ui/spinner_test.rb

```diff
@@ -60,19 +89,11 @@ def match_lines(out, *patterns)
         # newline, or cursor-down
         lines = out.split(/\n|\x1b\[\d*B/)
 
-        # Assert all lines are matched
-        lines.each do |l|
```

Comment:
I don't think this block make sense. I just changed it to match all lines one by one